src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [user, setUser] = useState<User | null>(null);
+  const [filterMethod, setFilterMethod] = useState('');
+  const [query, setQuery] = useState('');
+  const onClickHandler = (currentTodo: Todo | null) => {
+    setSelectedTodo(currentTodo);
+    setUser(null);
+  };
+
+  useEffect(() => {
+    getTodos().then(todosFromServer => {
+      setTodos(todosFromServer);
+    });
+  }, []);
+
+  useEffect(() => {
+    if (selectedTodo) {
+      getUser(selectedTodo.userId).then(userFromServer => {
+        setUser(userFromServer);
+      });
```

Comment:
```suggestion
      getUser(selectedTodo.userId)
        .then(setUser);
```

Reply:
not fixed

Reply:
Isn't it?
![image](https://github.com/user-attachments/assets/87739fa5-26b5-4005-85bf-0d80bef06331)
Mb your are looking at old commit? Because I fixed it